### ISSUE TITLE
Autoscale fails on negative numbers

### ISSUE THREAD

**Issue by tovrstra** [juhasch/PhysicalQuantities#112]
With `PhysicalQuantities-0.9.1` in IPython, I get:

```
>>> (-0.00032C).autoscale
/home/foo/.local/lib/python3.6/site-packages/PhysicalQuantities/quantity.py:580: RuntimeWarning: invalid value encountered in log10
  n = np.log10(b.value)
-0.00032 C
```

**Comment by juhasch** [comment]
Thanks for the report. Should be fixed in PR #115

**Comment by tovrstra** [comment]
Allright, looks good too.
